arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -1433,15 +1413,15 @@
       }
     },
     "ErrorDetails": {
-      "description": "The properties related to the details of an error.",
+      "description": "Error details.",
```

Comment:
1. Are you sure that all the model properties are actually PascalCase. Please check on Fiddler. Looking at the C# code in the service is not very helpful.